essays/AITBW-roadToDune/index.md

```diff
@@ -0,0 +1,93 @@
+# Dr. Strangelove or: How I handled my stubbornness to become a web developer
+
+Becoming a developer can be a pretty overwhelming task these days, especially if you delve into the [JavaScript](https://hackernoon.com/how-it-feels-to-learn-javascript-in-2016-d3a717dd577f) land, where _the new kid on the block_ can be thrown into obscurity in a matter of days.
```

Comment:
I think "delve into the Javascript land" sounds better as "delve into Javascript land". The "the" in that position sounds more like spanish than english. Spanish never omits "el" o "la", whereas english often omits "the" from a lot of places.